hotxlfp/formulas/operators.py

```diff
@@ -77,10 +78,10 @@ def __eq__(self, other):
         return self.value == other
 
     def __ge__(self, other):
-        return self.__gt__(other) or self.__eq__(other)
+        return torch.logical_or(self.__gt__(other), self.__eq__(other))
```

Comment:
does .__ge__ not exist on "self" and "other"?

Reply:
Not sure if I understand your question. I think this is the `__ge__` method of the ExcelComparator? It seems like this class just overrides `__gt__` and `__lt__`, and then overrides the equals version by reusing these functions.

Reply:
ah, I see.  I was wondering whether there were torch vectors that were passed in here.  that makes sense